epictrack-web/src/components/myTasks/MyTasksList.tsx

```diff
@@ -175,8 +178,11 @@ export default function MyTasksList() {
         Cell: canEdit
           ? ({ cell, row, renderedCellValue }) => (
               <ETGridTitle
-                to={`/work-plan?work_id=${row.original.work.id}`}
-                onClick={() => onEdit(row.original.id)}
+                to={`/my-tasks`}
+                onClick={() => {
+                  setTask(row.original);
+                  setShowModalForm(true);
```

Comment:
Can you consider improving the design so we don't have to do a setTask and a setShowModalForm separately